magda-web-client/src/config.ts

```diff
@@ -181,6 +185,18 @@ const vocabularyApiEndpoints =
               "https://vocabs.ands.org.au/repository/api/lda/ands-nc/controlled-vocabulary-for-resource-type-genres/version-1-1/concept.json"
           ];
 
+let defaultCkanServer: string;
+if (process.env.NODE_ENV === "development") {
```

Comment:
Do we want to make it configurable (through helm and web-server/server.config)?
@AlexGilleran